pkg/asset/machines/gcp/machines.go

```diff
@@ -70,7 +70,7 @@ func provider(clusterID string, platform *gcp.Platform, mpool *gcp.MachinePool,
 	az := mpool.Zones[azIdx]
 	return &gcpprovider.GCPMachineProviderSpec{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "gcpproviderconfig.openshift.io/v1beta1",
+			APIVersion: "gcpprovider.openshift.io/v1beta1",
```

Comment:
can you point to the docs/link which is the source of truth for this..?

Reply:
hmm maybe https://github.com/openshift/cluster-api-provider-gcp/blob/cb0305f3b986e67ff8d5bb0b375dd7bbb835db01/pkg/apis/gcpprovider/v1beta1/register.go#L6

Reply:
@abhinavdahiya yes, that's it